e2etests/package.json

```diff
@@ -56,8 +63,8 @@
     "module_methods": "mocha --reporter mocha-multi-reporters --reporter-options configFile=../mocha-reports.json test/module_methods.js",
     "flaky_tests": "npm-run-all -p -l  job_client",
     "phase0_modules": "npm-run-all -p -l module_crud module_messaging module_twin module_methods configurations",
-    "phase1_fast": "npm-run-all -p -l service registry device_acknowledge_tests upload_disconnect authentication method_disconnect",
-    "phase2_slow": "npm-run-all -p -l device_method twin_e2e_tests sas_token_tests device_service twin_disconnect c2d_disconnect d2c_disconnect",
```

Comment:
>twin_disconnect [](start = 98, length = 15)

please add the removed tests to the "flaky_test" phase

Reply:
Done in another codeflow.

---
In reply to: [221354064](https://github.com/Azure/azure-iot-sdk-node/pull/378#discussion_r221354064) [](ancestors = 221354064)